app/controllers/LandingController.scala

```diff
@@ -122,6 +125,26 @@ class LandingController @Inject()(val landingService: LandingService,
     }
   }
 
+  private def hasIncompleteResponsiblePeople()(implicit authContext: AuthContext, headerCarrier: HeaderCarrier): Future[Boolean] = {
+
+    statusService.getDetailedStatus.flatMap {
+      case (SubmissionDecisionRejected |
+            SubmissionDecisionRevoked |
+            DeRegistered |
+            SubmissionDecisionExpired |
+            SubmissionWithdrawn, _) =>
```

Comment:
you could possibly do this check in StatusService, there are already implemented methods like isPreSubmission, isPending so it seems to be reasonable to do this there, but I'm happy to accept your approach as well

Reply:
you are right, we could/should place it there.
but for the time left in this sprint, lets defer this to future code changes.